### ISSUE TITLE
Failing deploy due to Resource name not being alphanumeric

### ISSUE THREAD

**Issue by sung-hwang-zocdoc** [serverless/serverless#1924]
* Check if an Issue already exists for this
* Add the relevant label(s)
* Be kind :)
Not sure if this is an expected behavior, but this had not occurred in version 0.5. We have dots or dashes in our S3 buckets and it's failing due to buckets not being alphanumeric.

Bug Report Template -----------------------------------------------

##### Serverless Framework Version: 
1.0.0-beta.2
##### Operating System: 
Windows (Powershell)
##### Expected Behavior: 

Had events definition in serverless.yml as:
    events:
      - s3:
          name: myEvent
          bucket: org.abc.searchhistory
          event: s3:ObjectCreated:*

We had the same bucket configured in version 0.5. and worked just fine. Was expecting the same with beta version.

##### Actual Behavior:
But it fails to deploy with below message, saying resource isn't alphanumeric.
When I changed the s3 bucket name excluding dots, it was successful.

##### Stack-Trace (Optional):
Here's what it returns:

Serverless: Zipping service...
Serverless: Removing old service versions...
Serverless: Uploading .zip file to S3...
Serverless: Updating Stack...

  Serverless Error ---------------------------------------

     Template format error: Resource name org.abc.searchhistory0S3Event
     is non alphanumeric.

Feature Request Template ------------------------------------------

##### Benefits:

* Benefit one...

##### Drawbacks:

* Drawback one...

##### Additional Details:

Discussion template  ----------------------------------------------

##### TLDR: 
Deploy failing due to Resource name being non alphanumeric. Expected?

**Comment by flomotlik** [comment]
Thanks for reporting @sung-hwang-zocdoc we're working on that naming in #1869 and will solve this issue with it as well.

**Comment by flomotlik** [comment]
This is resolved with #1951
